twilight-gateway/src/lib.rs

```diff
@@ -34,7 +34,7 @@ mod tls;
 #[cfg(any(feature = "zlib-stock", feature = "zlib-simd"))]
 pub use self::inflater::Inflater;
 pub use self::{
-    channel::MessageSender,
+    channel::MessageQueue,
```

Comment:
I think CI remarks MessageSender as dead code due to it no longer being public (reexported from lib.rs)

Reply:
I will also export the `MessageSender` typealias then.

Reply:
Resolved as of latest commit.